### PR TITLE
Removed prerender code.

### PR DESCRIPTION
**Pull request by D-Bits**
Included JSX prerender code.

### CODE REVIEW

File: democracylab/settings.py [hunk 1]
```diff
@@ -100,8 +99,8 @@
 
 # TODO: Use     'django_seo_js.middleware.UserAgentMiddleware',
 MIDDLEWARE = [
-    'common.helpers.caching.DebugUserAgentMiddleware',
     'django.middleware.security.SecurityMiddleware',
+    'django_seo_js.middleware.UserAgentMiddleware',
```
Comment (marlonkeating) [inline democracylab/settings.py:103]: Should remove this and the dependency in https://github.com/DemocracyLab/CivicTechExchange/blob/4b420b49872edf9fc3a87828f829ccbba65eece0/requirements.txt#L32

File: democracylab/settings.py [hunk 2]
```diff
@@ -215,7 +214,7 @@ def read_connection_config(config):
 EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
 
 PROTOCOL_DOMAIN = os.environ['PROTOCOL_DOMAIN']
-ADMIN_EMAIL = os.environ['ADMIN_EMAIL']
+ADMIN_EMAIL = os.getenv('ADMIN_EMAIL')
```
Comment (marlonkeating) [inline democracylab/settings.py:217]: Any reason for this change?  ADMIN_EMAIL is a required environment variable, so it's okay if a null variable throws an exception

File: democracylab/settings.py [hunk 3]
```diff
@@ -400,9 +399,6 @@ def read_connection_config(config):
 
 # If you're using prerender.io (the default backend):
 SEO_JS_PRERENDER_TOKEN = os.environ.get('SEO_JS_PRERENDER_TOKEN', '')
-SEO_JS_BACKEND = "common.helpers.caching.DebugPrerenderIO"
```
Comment (marlonkeating) [inline democracylab/settings.py:403]: Anything prefixed with SEO_JS_ is something we should be able to get rid of.

**Review by marlonkeating — CHANGES_REQUESTED** [review]

File: democracylab/settings.py (hunk 3 above)
Comment (D-Bits) [reply democracylab/settings.py:403]: I got a `ValueError: Missing SEO_JS_PRERENDER_TOKEN in settings.` error when I tried to boot the containers after removing those values.

Comment (marlonkeating) [reply democracylab/settings.py:403]: Did you try it after removing the library references from requirements.txt and MIDDLEWARE?

**Pull request closed without merging**